packages/nimbus/src/components/tooltip/components/tooltip.content.tsx

```diff
@@ -0,0 +1,34 @@
+import { Tooltip as RATooltip } from "react-aria-components";
+import { TooltipRoot } from "../tooltip.slots";
```

Comment:
Since theres a root slot and a root component, it might be good to update the slot name to reduce confusion?

```suggestion
import { TooltipRootSlot } from "../tooltip.slots";
```

Reply:
Thanks! 38c64d6a70fbc53414a85661433f40eb89c85fd2